internal/bminventory/inventory.go

```diff
@@ -187,6 +187,15 @@ const ignitionConfigFormat = `{
       },
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
 	},
+    {
```

Comment:
Shouldn't we add this only if IIPv6 is used?

Reply:
We don't know if it is IPv6 only.  After discovery we know